cmd/spicedb/servetesting_integration_test.go

```diff
@@ -39,10 +40,18 @@ func TestTestServer(t *testing.T) {
 	require.NoError(err)
 	defer conn.Close()
 
+	resp, err := healthpb.NewHealthClient(conn).Check(context.Background(), &healthpb.HealthCheckRequest{Service: "authzed.api.v1alpha1.SchemaService"})
+	require.NoError(err)
+	require.Equal(healthpb.HealthCheckResponse_SERVING, resp.GetStatus())
```

Comment:
you could also use https://pkg.go.dev/github.com/stretchr/testify/require#Eventually here to so that the test doesn't fail if it happens to hit the server before it's ready

Reply:
@ecordell See my comment above